### DISCUSSION TITLE
2024/todo/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/shane0.github.io#51] [General]
# 2024/todo/

https://shanenull.com/2024/todo/

**Comment by shane0** [comment]
screecast about inherent software problems reference joe armstrong the mess and the node guys video about issues with node & linters to detect cognitave collapse in software platforms and hardware designs  https://shanenull.com/2023/collapse/collapse/#collapse-of-complex-systems

**Reply by shane0** [reply]
https://www.vimonlineeditor.com/

**Reply by shane0** [reply]
https://www.up.com/heritage/steam/index.htm

**Comment by shane0** [comment]
https://shanenull.com/workflow/todo/

**Reply by shane0** [reply]
http://shanenull.com/2024/todo/
